package.json

```diff
@@ -45,5 +46,10 @@
     "test": "forge test",
     "test:coverage": "forge coverage",
     "test:coverage:report": "forge coverage --report lcov && genhtml lcov.info --branch-coverage --output-dir coverage"
+  },
+  "module": "index.ts",
+  "type": "module",
+  "peerDependencies": {
+    "typescript": "^5.0.0"
   }
-}
+}
```

Comment:
Again, do we really need this here? Did you have any difficulties with running Bun?

Reply:
Yes I had difficulties running bun